lightning/src/ln/channelmanager.rs

```diff
@@ -3173,8 +3174,11 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 		}
 	}
 
-	/// We force-close the channel without letting our counterparty participate in the shutdown
-	fn block_disconnected(&self, header: &BlockHeader, _: u32) {
+	/// Updates channel state based on a disconnected block.
+	///
+	/// If necessary, the channel may be force-closed without letting the counterparty participate
+	/// in the shutdown.
```

Comment:
What does this mean ? IIRC the only case where you may have to force-close a channel due to disconnection is because the funding is disconnect, so it can't imply interacting with the counterparty as there is not mutual closing tx to agree on ?

Reply:
Right we mean the same thing "without interacting with the counterparty" is clearer as `shutdown` is a well-defined LN message :) But I'm not even sure we can qualify this of a force-close like when we broadcast the commitment. We just forget about the channel.

Reply:
We return a `ChannelMonitorUpdateStep::ChannelForceClosed` which is going to be send to chain_monitor. The first and (last) commitment transaction will try to be broadcast, which should result in a failure as the funding transaction isn't confirmed. I don't think we should worry about this outcome.

Reply:
Do you have a specific recommendation here? (e.g., remove the sentence entirely)